clients/vscode/src/FileTrackProvider.ts

```diff
@@ -0,0 +1,69 @@
+import { EditorParams } from "tabby-agent";
+import { TextEditor, window } from "vscode";
+
+export class FileTrackerProvider {
```

Comment:
It seems this class is not used as the methods in this class do not need private context.  
Let's export the two functions directly. 
```
export function collectVisibleEditors ...
export function collectActiveEditor ...
```
The file name can be changed to `windowUtils`